### PR TITLE
Editorial: Add Bikeshed metadata for 'request'

### PR DESCRIPTION
**Pull request by mikewest**
See tabatkins/bikeshed#773 for context.

### CODE REVIEW

**Comment by mikewest** [comment]
Moar bikeshedding.

File: Overview.src.html [hunk 1]
```diff
@@ -674,8 +677,9 @@ <h4 id=terminology-headers>Headers</h4>
 <span title=concept-request-window>window</span>.
 
 <p>A <span title=concept-request>request</span> has an associated
-<dfn title=concept-request-target-browsing-context>target browsing context</dfn> (null or a
-<span data-anolis-spec=html>browsing context</span>). Unless stated otherwise it is null.
+<dfn title=concept-request-target-browsing-context data-export data-dfn-for=request>target browsing
+  context</dfn> (null or a <span data-anolis-spec=html>browsing context</span>). Unless stated
```
Comment (annevk) [inline Overview.src.html]: no newlines inside an "inline" element, in that case it's fine to go beyond 100cols

File: Overview.src.html [hunk 2]
```diff
@@ -940,14 +951,14 @@ <h4 id=terminology-headers>Headers</h4>
 for other values. If <cite>HTML</cite> changes here, this standard will need corresponding changes.
 
 <p>A <span title=concept-request>request</span> has an associated
-<dfn title=concept-request-use-url-credentials-flag>use-URL-credentials flag</dfn>.
-Unless stated otherwise, it is unset.
+<dfn title=concept-request-use-url-credentials-flag data-export data-dfn-for=request>use-URL-credentials
+flag</dfn>. Unless stated otherwise, it is unset.
```
Comment (annevk) [inline Overview.src.html]: same

File: Overview.src.html [hunk 3]
```diff
@@ -998,22 +1009,22 @@ <h4 id=terminology-headers>Headers</h4>
...
-<dfn title=concept-request-redirect-mode>redirect mode</dfn>, which is
+<dfn title=concept-request-redirect-mode data-export data-dfn-for=request>redirect mode</dfn>, which is
 "<code title>follow</code>", "<code title>error</code>", or "<code title>manual</code>".
 Unless stated otherwise, it is "<code title>follow</code>".
 
 <p>A <span title=concept-request>request</span> has associated
-<dfn title=concept-request-integrity-metadata>integrity metadata</dfn> (a string). Unless
-stated otherwise, it is the empty string.
+<dfn title=concept-request-integrity-metadata data-export data-dfn-for=request>integrity
+metadata</dfn> (a string). Unless stated otherwise, it is the empty string.
```
Comment (annevk) [inline Overview.src.html]: same

File: Overview.src.html [hunk 4]
```diff
@@ -998,22 +1009,22 @@ <h4 id=terminology-headers>Headers</h4>
...
 
 <p>A <span title=concept-request>request</span> has associated
-<dfn title=concept-request-nonce-metadata>cryptographic nonce metadata</dfn> (a string). Unless
-stated otherwise, it is the empty string.
+<dfn title=concept-request-nonce-metadata data-export data-dfn-for=request>cryptographic nonce
+metadata</dfn> (a string). Unless stated otherwise, it is the empty string.
```
Comment (annevk) [inline Overview.src.html]: same

File: Overview.src.html [hunk 5]
```diff
@@ -1025,27 +1036,27 @@ <h4 id=terminology-headers>Headers</h4>
...
-it is zero.
+<dfn title=concept-request-redirect-count data-export data-dfn-for=request>redirect count</dfn>.
+Unless stated otherwise, it is zero.
 
 <p>A <span title=concept-request>request</span> has an associated
-<dfn title=concept-request-response-tainting>response tainting</dfn>, which is
-"<code title>basic</code>", "<code title>cors</code>", or "<code title>opaque</code>".
-Unless stated otherwise, it is "<code title>basic</code>".
+<dfn title=concept-request-response-tainting data-export data-dfn-for=request>response
+tainting</dfn>, which is "<code title>basic</code>", "<code title>cors</code>", or
```
Comment (annevk) [inline Overview.src.html]: same

File: Overview.src.html [hunk 6]
```diff
@@ -1113,57 +1124,59 @@ <h4 id=terminology-headers>Headers</h4>
 Unless stated otherwise, it is "<code title>default</code>".
 
 <p>A <span title=concept-response>response</span> can have an associated
-<dfn title=concept-response-termination-reason>termination reason</dfn> which is
-<i title>end-user abort</i>, <i title>fatal</i>, or <i title>timeout</i>.
+<dfn title=concept-response-termination-reason data-export data-dfn-for=response>termination
+reason</dfn> which is <i title>end-user abort</i>, <i title>fatal</i>, or <i title>timeout</i>.
```
Comment (annevk) [inline Overview.src.html]: same

File: Overview.src.html [hunk 7]
```diff
@@ -1113,57 +1124,59 @@ <h4 id=terminology-headers>Headers</h4>
...
+<dfn title=concept-response-csp-list data-export data-dfn-for=response>CSP list</dfn>, which is a
+list of <a href=https://w3c.github.io/webappsec-csp/#policy>Content Security Policy objects</a>
 for the <span title=concept-response>response</span>. The list is empty unless otherwise
 specified. <span data-anolis-ref>CSP</span>
 
 <p>A <span title=concept-response>response</span> has an associated
-<dfn title=concept-response-cors-exposed-header-name-list>CORS-exposed header-name list</dfn> (a
-list of zero or more <span title=concept-header>header</span>
+<dfn title=concept-response-cors-exposed-header-name-list data-export data-dfn-for=response>CORS-exposed header-name
+list</dfn> (a list of zero or more <span title=concept-header>header</span>
```
Comment (annevk) [inline Overview.src.html]: same

File: Overview.src.html [hunk 8]
```diff
@@ -1246,8 +1259,8 @@ <h4 id=terminology-headers>Headers</h4>
 <span title=concept-response-body>body</span> is null, and
 <span title=concept-response-trailer>trailer</span> is empty.
 
-<p>An <dfn title=concept-filtered-response-opaque-redirect>opaque-redirect filtered response</dfn>
-is a <span title=concept-filtered-response>filtered response</span> whose
+<p>An <dfn title=concept-filtered-response-opaque-redirect data-export>opaque-redirect filtered
+response</dfn> is a <span title=concept-filtered-response>filtered response</span> whose
```
Comment (annevk) [inline Overview.src.html]: same

Comment (annevk) [inline Overview.src.html]: break after "An" though

File: Overview.src.html [hunk 9]
```diff
@@ -1294,8 +1307,8 @@ <h4 id=terminology-headers>Headers</h4>
 
 <h3>Authentication entries</h3>
 
-<p>An <dfn>authentication entry</dfn> and a <dfn>proxy-authentication entry</dfn> are
-tuples of username, password, and realm, associated with one or more
+<p>An <dfn data-export>authentication entry</dfn> and a <dfn data-export>proxy-authentication
+entry</dfn> are tuples of username, password, and realm, associated with one or more
```
Comment (annevk) [inline Overview.src.html]: same

File: Overview.src.html [hunk 10]
```diff
@@ -1335,14 +1348,14 @@ <h4 id=terminology-headers>Headers</h4>
 
 <h3>Connections</h3>
 
-<p>A user agent has an associated <dfn title=concept-connection-pool>connection pool</dfn>. A
-<span title=concept-connection-pool>connection pool</span> consists of zero or more
-<dfn title=concept-connection>connections</dfn>. Each
+<p>A user agent has an associated <dfn title=concept-connection-pool data-export>connection
+pool</dfn>. A <span title=concept-connection-pool>connection pool</span> consists of zero or more
```
Comment (annevk) [inline Overview.src.html]: same

File: Overview.src.html [hunk 11]
```diff
@@ -1377,7 +1390,8 @@ <h4 id=terminology-headers>Headers</h4>
 <h3>Port blocking</h3>
 
 <p>To determine whether fetching a <span title=concept-request>request</span> <var>request</var>
-<dfn title="block bad port">should be blocked due to a bad port</dfn>, run these steps:
+<dfn title="block bad port" data-export data-lt="block bad port">should be blocked due to a bad
+port</dfn>, run these steps:
```
Comment (annevk) [inline Overview.src.html]: same

File: Overview.src.html [hunk 12]
```diff
@@ -1398,8 +1412,8 @@ <h4 id=terminology-headers>Headers</h4>
  <li><p>Return <b>allowed</b>.
 </ol>
 
-<p>A <span data-anolis-spec=url title=concept-url-port>port</span> is a <dfn>bad port</dfn> if it
-is listed in the first column of the following table.
+<p>A <span data-anolis-spec=url title=concept-url-port>port</span> is a <dfn data-export>bad
+port</dfn> if it is listed in the first column of the following table.
```
Comment (annevk) [inline Overview.src.html]: same

File: Overview.src.html [hunk 13]
```diff
@@ -1492,21 +1506,21 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
...
-  "unsafe-url"
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>no-referrer</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>no-referrer-when-downgrade</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>same-origin</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>origin</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>strict-origin</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>origin-when-cross-origin</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>strict-origin-when-cross-origin</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>unsafe-url</dfn>"
 };</pre>
```
Comment (annevk) [inline Overview.src.html]: I thought this kind of thing would happen automatically. Also, does anyone need to reference this?

File: Overview.src.html [hunk 14]
```diff
@@ -1532,27 +1546,27 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
...
 
-<p>A <dfn title=concept-ReadableStream>ReadableStream</dfn> object represents a
-<span data-anolis-spec=streams title=readablestream>stream of data</span>. In this section, we
-define common operations for ReadableStream. <span data-anolis-ref>STREAMS</span>
+<p>A <dfn title=concept-ReadableStream data-export data-dfn-type=interface>ReadableStream</dfn>
+object represents a <span data-anolis-spec=streams title=readablestream>stream of data</span>. In
+this section, we define common operations for ReadableStream. <span data-anolis-ref>STREAMS</span>
 
-<p>To <dfn title=concept-enqueue-ReadableStream>enqueue</dfn> <var>chunk</var> into a
+<p>To <dfn title=concept-enqueue-ReadableStream data-export data-dfn-for=ReadableStream>enqueue</dfn> <var>chunk</var> into a
```
Comment (annevk) [inline Overview.src.html]: 100 col

File: Overview.src.html [hunk 15]
```diff
@@ -1561,9 +1575,10 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
  <var>reason</var>). Rethrow any exceptions.
 </ol>
 
-<p>To <dfn title=concept-construct-ReadableStream>construct</dfn> a
-<code title=concept-ReadableStream>ReadableStream</code> object with given <var>strategy</var>,
-<var>pull</var> action and <var>cancel</var> action, all of which are optional, run these steps:
+<p>To <dfn title=concept-construct-ReadableStream data-export data-dfn-for=ReadableStream>construct
+a <code title=concept-ReadableStream>ReadableStream</code> object</dfn> with given
```
Comment (annevk) [inline Overview.src.html]: no newlines

File: Overview.src.html [hunk 16]
```diff
@@ -1581,9 +1596,10 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
  <li><p>Return <var>stream</var>.
 </ol>
 
-<p>To <dfn title=concept-construct-fixed-ReadableStream>construct</dfn> a fixed
-<code title=concept-ReadableStream>ReadableStream</code> object with given <var>chunks</var>,
-run these steps:
+<p>To
+<dfn title=concept-construct-fixed-ReadableStream data-export data-dfn-for=ReadableStream>construct
+a fixed <code title=concept-ReadableStream>ReadableStream</code> object</dfn> with given
```
Comment (annevk) [inline Overview.src.html]: no newlines

File: Overview.src.html [hunk 17]
```diff
@@ -1611,14 +1628,15 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
  <li><p>Return <var>reader</var>.
 </ol>
 
-<p>To <dfn title=concept-read-chunk-from-ReadableStream>read a chunk</dfn> from a
-<code title=concept-ReadableStream>ReadableStream</code> object with <var>reader</var>, return the
-result of calling
+<p>To <dfn title=concept-read-chunk-from-ReadableStream data-export data-dfn-for=ReadableStream>read
+a chunk</dfn> from a <code title=concept-ReadableStream>ReadableStream</code> object with
```
Comment (annevk) [inline Overview.src.html]: no newlines

File: Overview.src.html [hunk 18]
```diff
@@ -1611,14 +1628,15 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
...
+<var>reader</var>, return the result of calling
 <code data-anolis-spec=streams>ReadFromReadableStreamReader</code>(<var>reader</var>).
 
-<p>To <dfn title=concept-read-all-bytes-from-ReadableStream>read all bytes</dfn> from a
-<code title=concept-ReadableStream>ReadableStream</code> object with <var>reader</var>, run these
-steps:
+<p>To
+<dfn title=concept-read-all-bytes-from-ReadableStream data-export data-dfn-for=ReadableStream>read
+all bytes</dfn> from a <code title=concept-ReadableStream>ReadableStream</code> object with
```
Comment (annevk) [inline Overview.src.html]: no newlines

File: Overview.src.html [hunk 19]
```diff
@@ -1675,23 +1693,25 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
...
-<code data-anolis-spec=streams>IsReadableStreamLocked</code>(<var>stream</var>) is true.
+<dfn title=concept-ReadableStream-locked data-export data-dfn-for=ReadableStream>locked</dfn> if the
+result of calling <code data-anolis-spec=streams>IsReadableStreamLocked</code>(<var>stream</var>) is
+true.
 
 <p>A <code title=concept-ReadableStream>ReadableStream</code> object <var>stream</var> is said to
-<dfn title=concept-ReadableStream-need-more-data>need more data</dfn> if the following conditions
-hold:
+<dfn title=concept-ReadableStream-need-more-data data-export data-dfn-for=ReadableStream>need more
+data</dfn> if the following conditions hold:
```
Comment (annevk) [inline Overview.src.html]: no newlines

File: Overview.src.html [hunk 20]
```diff
@@ -2306,8 +2329,8 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 
 <h3>Main fetch</h3>
 
-<p>To perform a <dfn title=concept-main-fetch>main fetch</dfn> using <var>request</var>, optionally
-with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
+<p>To perform a <dfn title=concept-main-fetch data-export>main fetch</dfn> using <var>request</var>,
+optionally with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
```
Comment (annevk) [inline Overview.src.html]: I don't think we should export main fetch.

File: Overview.src.html [hunk 21]
```diff
@@ -2636,7 +2659,7 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 
 <h3>Basic fetch</h3>
 
-<p>To perform a <dfn title=concept-basic-fetch>basic fetch</dfn> using
+<p>To perform a <dfn title=concept-basic-fetch data-export>basic fetch</dfn> using
```
Comment (annevk) [inline Overview.src.html]: Same here. Nobody should invoke this from the outside.

File: Overview.src.html [hunk 22]
```diff
@@ -2757,7 +2780,7 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 
 <h3>HTTP fetch</h3>
 
-<p>To perform an <dfn title=concept-http-fetch>HTTP fetch</dfn> using
+<p>To perform an <dfn title=concept-http-fetch data-export>HTTP fetch</dfn> using
```
Comment (annevk) [inline Overview.src.html]: Same.

File: Overview.src.html [hunk 23]
```diff
@@ -2952,7 +2975,7 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 in addition to <span title=concept-http-fetch>HTTP fetch</span> above.
 <span data-anolis-ref>HTML</span>
 
-<p>To perform an <dfn title=concept-http-redirect-fetch>HTTP-redirect fetch</dfn> using
+<p>To perform an <dfn title=concept-http-redirect-fetch data-export>HTTP-redirect fetch</dfn> using
```
Comment (annevk) [inline Overview.src.html]: This one is okay, but only HTML needs to and it doesn't use bikeshed.

File: Overview.src.html [hunk 24]
```diff
@@ -3034,7 +3057,7 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 <h3>HTTP-network-or-cache fetch</h3>
 
 <p>To perform an
-<dfn title=concept-http-network-or-cache-fetch>HTTP-network-or-cache fetch</dfn> using
+<dfn title=concept-http-network-or-cache-fetch data-export>HTTP-network-or-cache fetch</dfn> using
```
Comment (annevk) [inline Overview.src.html]: Also no external invocation allowed.

File: Overview.src.html [hunk 25]
```diff
@@ -3388,7 +3411,7 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 
 <h3>HTTP-network fetch</h3>
 
-<p>To perform an <dfn title=concept-http-network-fetch>HTTP-network fetch</dfn> using
+<p>To perform an <dfn title=concept-http-network-fetch data-export>HTTP-network fetch</dfn> using
```
Comment (annevk) [inline Overview.src.html]: No externals.

File: Overview.src.html [hunk 26]
```diff
@@ -3766,28 +3789,33 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 
 <h3>CORS-preflight cache</h3>
```
Comment (annevk) [inline Overview.src.html]: Most of this shouldn't be used externally either.

File: Overview.src.html (hunk 1 above)
Comment (mikewest) [inline Overview.src.html]: Ok, got it. I should have looked at the examples; I read "Do not use newlines inside "inline" elements" as opposed to HTML's rules which force breaks inside the opening tag.

File: Overview.html [hunk 27]
```diff
@@ -3957,7 +3957,7 @@ <h3 id="cors-preflight-cache"><span class="secno">5.8 </span>CORS-preflight cach
 
 <h3 id="cors-check"><span class="secno">5.9 </span>CORS check</h3>
 
-<p>To perform a <dfn id="concept-cors-check" title="concept-cors-check">CORS check</dfn> for a
+<p>To perform a <dfn data-export="" id="concept-cors-check" title="concept-cors-check">CORS check</dfn> for a
```
Comment (annevk) [inline Overview.html]: No external users here either.

File: Overview.html [hunk 28]
```diff
@@ -4082,7 +4082,7 @@ <h3 id="headers-class"><span class="secno">6.1 </span>Headers class</h3>
 
 <pre class="idl">[<a href="#dom-headers" title="dom-Headers">Constructor</a>(optional <a href="#headersinit">HeadersInit</a> <var>init</var>),
  Exposed=(Window,Worker)]
-interface <dfn id="headers">Headers</dfn> {
+interface <dfn data-dfn-type="interface" data-export="" id="headers">Headers</dfn> {
```
Comment (annevk) [inline Overview.html]: You used to have these attributes at the end.

File: Overview.src.html [hunk 29]
```diff
@@ -4113,8 +4114,9 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   </ol>
 </ol>
 
-<p>The <dfn title=dom-Headers><code>Headers(<var>init</var>)</code></dfn> constructor,
-when invoked, must run these steps:
+<p>The
+<dfn title=dom-Headers data-export data-dfn-for=Headers data-dfn-type=constructor data-lt="Headers(init)"><code>Headers(<var>init</var>)</code></dfn>
```
Comment (annevk) [inline Overview.src.html]: Is that `lt` really needed?

Comment (annevk) [inline Overview.src.html]: Also, constructors and methods generally shouldn't be referenced elsewhere. Maybe in examples and such...

File: Overview.src.html (hunk 13 above)
Comment (mikewest) [inline Overview.src.html]: > I thought this kind of thing would happen automatically.

It doesn't look like Anolis does much IDL processing: https://fetch.spec.whatwg.org/#referrerpolicy

> Also, does anyone need to reference this?

I'd suggest that that's the wrong way to look at it. If it's important enough to define in the document, it's going to be interesting to someone somewhere. *shrug*

Comment (annevk) [inline Overview.src.html]: Fair I guess. Though there has been some talk of moving this into Referrer Policy, I'm not objecting to taking this in for now.

File: Overview.src.html (hunk 20 above)
Comment (mikewest) [inline Overview.src.html]: Ok. I poke at many of these (though non-normatively) from CSP to explain the call flow for various algorithms, but I can do those as one-offs if you don't want to export it generally.

Comment (annevk) [inline Overview.src.html]: I guess it's fine to export in that case. Hopefully nobody ever considers implementing a public API for these.

File: Overview.src.html (hunk 29 above)
Comment (mikewest) [inline Overview.src.html]: > Is that `lt` really needed?

I don't know, honestly. It's not clear to me whether the backend needs a clean `lt`, or whether it parses out data from the tag, even if the value includes HTML. *shrug* @tabatkins would know, I'm sure.

> Also, constructors and methods generally shouldn't be referenced elsewhere. Maybe in examples and such...

As above, I'd err on the side of exporting everything, but _I_ don't personally need any of this, so...

Comment (annevk) [inline Overview.src.html]: I think it can deal with the markup just fine.

File: Overview.html [hunk 30]
```diff
@@ -1378,9 +1378,9 @@ <h4 id="responses"><span class="secno">3.1.6 </span>Responses</h4>
 
 <h3 id="authentication-entries"><span class="secno">3.2 </span>Authentication entries</h3>
 
-<p>An <dfn data-export="" id="authentication-entry">authentication entry</dfn> and a <dfn data-export="" id="proxy-authentication-entry">proxy-authentication
-entry</dfn> are tuples of username, password, and realm, associated with one or more
-<a href="#concept-request" title="concept-request">requests</a>.
+<p>An <dfn data-export="" id="authentication-entry">authentication entry</dfn> and a
+<dfn data-export="" id="proxy-authentication-entry">proxy-authentication entry</dfn> are tuples of username, password, and realm,
```
Comment (annevk) [inline Overview.html]: 100 col

File: Overview.html [hunk 31]
```diff
@@ -1419,8 +1419,9 @@ <h3 id="fetch-groups"><span class="secno">3.3 </span>Fetch groups</h3>
 
 <h3 id="connections"><span class="secno">3.4 </span>Connections</h3>
 
-<p>A user agent has an associated <dfn data-export="" id="concept-connection-pool" title="concept-connection-pool">connection
-pool</dfn>. A <a href="#concept-connection-pool" title="concept-connection-pool">connection pool</a> consists of zero or more
+<p>A user agent has an associated
+<dfn data-export="" id="concept-connection-pool" title="concept-connection-pool">connection pool</dfn>. A
```
Comment (annevk) [inline Overview.html]: 100 col

File: Overview.html [hunk 32]
```diff
@@ -1419,8 +1419,9 @@ <h3 id="fetch-groups"><span class="secno">3.3 </span>Fetch groups</h3>
...
+<a href="#concept-connection-pool" title="concept-connection-pool">connection pool</a> consists of zero or more
```
Comment (annevk) [inline Overview.html]: 100 col

File: Overview.html [hunk 33]
```diff
@@ -5597,9 +5623,6 @@ <h2 class="no-num" id="references">References</h2>
 <dt id="refsMIX">[MIX]
 <dd><cite><a href="https://w3c.github.io/webappsec-mixed-content/">Mixed Content</a></cite>, Mike West. W3C.
 
-<dt id="refsORIGIN">[ORIGIN]
-<dd>(Non-normative) <cite><a href="https://tools.ietf.org/html/rfc6454">The Web Origin Concept</a></cite>, Adam Barth. IETF.
```
Comment (annevk) [inline Overview.html]: Why did this happen?

File: Overview.src.html [hunk 34]
```diff
@@ -1492,21 +1507,21 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
...
-  "same-origin",
-  "origin",
-  "strict-origin",
-  "origin-when-cross-origin",
-  "strict-origin-when-cross-origin",
-  "unsafe-url"
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>no-referrer</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>no-referrer-when-downgrade</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>same-origin</dfn>",
+  "<dfn data-dfn-for=ReferrerPolicy data-dfn-type=enum-value data-export>origin</dfn>",
```
Comment (annevk) [inline Overview.src.html]: This might be what is confusing it somehow?

Comment (mikewest) [inline Overview.src.html]: ... Ah, build systems.

I'll just drop these `<dfn>` on the assumption that https://github.com/w3c/webappsec-referrer-policy/issues/62 pulls them into that document.

Comment (mikewest) [inline Overview.src.html]: Done in latest.

**Comment by annevk** [comment]
Thanks!

**Comment by annevk** [comment]
FYI, since you have write access you can create a branch directly on this master repository. That would allow us to collaborate on a PR more easily.

**Pull request merged**